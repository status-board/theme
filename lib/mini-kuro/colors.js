"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = {
  text: '#fff',
  textHighlight: '#fff',
  icon: '#fff',
  unknown: '#c0ab7f',
  success: '#4eb6a3',
  warning: '#d1be65',
  failure: '#ff9176'
};
exports["default"] = _default;
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uLy4uL3NyYy9taW5pLWt1cm8vY29sb3JzLnRzIl0sIm5hbWVzIjpbInRleHQiLCJ0ZXh0SGlnaGxpZ2h0IiwiaWNvbiIsInVua25vd24iLCJzdWNjZXNzIiwid2FybmluZyIsImZhaWx1cmUiXSwibWFwcGluZ3MiOiI7Ozs7OztlQUFlO0FBQ1hBLEVBQUFBLElBQUksRUFBRSxNQURLO0FBRVhDLEVBQUFBLGFBQWEsRUFBRSxNQUZKO0FBR1hDLEVBQUFBLElBQUksRUFBRSxNQUhLO0FBSVhDLEVBQUFBLE9BQU8sRUFBRSxTQUpFO0FBS1hDLEVBQUFBLE9BQU8sRUFBRSxTQUxFO0FBTVhDLEVBQUFBLE9BQU8sRUFBRSxTQU5FO0FBT1hDLEVBQUFBLE9BQU8sRUFBRTtBQVBFLEMiLCJzb3VyY2VzQ29udGVudCI6WyJleHBvcnQgZGVmYXVsdCB7XG4gICAgdGV4dDogJyNmZmYnLFxuICAgIHRleHRIaWdobGlnaHQ6ICcjZmZmJyxcbiAgICBpY29uOiAnI2ZmZicsXG4gICAgdW5rbm93bjogJyNjMGFiN2YnLFxuICAgIHN1Y2Nlc3M6ICcjNGViNmEzJyxcbiAgICB3YXJuaW5nOiAnI2QxYmU2NScsXG4gICAgZmFpbHVyZTogJyNmZjkxNzYnLFxufTtcbiJdfQ==