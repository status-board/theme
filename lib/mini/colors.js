"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _default = {
  text: '#000',
  textHighlight: '#000',
  icon: '#000',
  unknown: '#c0ab7f',
  success: '#4eb6a3',
  warning: '#d1be65',
  failure: '#ff9176'
};
exports["default"] = _default;
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uLy4uL3NyYy9taW5pL2NvbG9ycy50cyJdLCJuYW1lcyI6WyJ0ZXh0IiwidGV4dEhpZ2hsaWdodCIsImljb24iLCJ1bmtub3duIiwic3VjY2VzcyIsIndhcm5pbmciLCJmYWlsdXJlIl0sIm1hcHBpbmdzIjoiOzs7Ozs7ZUFBZTtBQUNYQSxFQUFBQSxJQUFJLEVBQUUsTUFESztBQUVYQyxFQUFBQSxhQUFhLEVBQUUsTUFGSjtBQUdYQyxFQUFBQSxJQUFJLEVBQUUsTUFISztBQUlYQyxFQUFBQSxPQUFPLEVBQUUsU0FKRTtBQUtYQyxFQUFBQSxPQUFPLEVBQUUsU0FMRTtBQU1YQyxFQUFBQSxPQUFPLEVBQUUsU0FORTtBQU9YQyxFQUFBQSxPQUFPLEVBQUU7QUFQRSxDIiwic291cmNlc0NvbnRlbnQiOlsiZXhwb3J0IGRlZmF1bHQge1xuICAgIHRleHQ6ICcjMDAwJyxcbiAgICB0ZXh0SGlnaGxpZ2h0OiAnIzAwMCcsXG4gICAgaWNvbjogJyMwMDAnLFxuICAgIHVua25vd246ICcjYzBhYjdmJyxcbiAgICBzdWNjZXNzOiAnIzRlYjZhMycsXG4gICAgd2FybmluZzogJyNkMWJlNjUnLFxuICAgIGZhaWx1cmU6ICcjZmY5MTc2Jyxcbn07XG4iXX0=