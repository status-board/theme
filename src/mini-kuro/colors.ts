export default {
    text: '#fff',
    textHighlight: '#fff',
    icon: '#fff',
    unknown: '#c0ab7f',
    success: '#4eb6a3',
    warning: '#d1be65',
    failure: '#ff9176',
};
