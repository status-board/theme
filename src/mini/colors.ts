export default {
    text: '#000',
    textHighlight: '#000',
    icon: '#000',
    unknown: '#c0ab7f',
    success: '#4eb6a3',
    warning: '#d1be65',
    failure: '#ff9176',
};
