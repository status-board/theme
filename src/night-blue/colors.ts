export default {
    text: '#eedba5',
    textHighlight: '#f6ecd0',
    background: '#1e2430',
    icon: '#e0c671',
    unknown: '#495b71',
    success: '#4ec2b4',
    warning: '#d1be65',
    failure: '#de5029',
};
