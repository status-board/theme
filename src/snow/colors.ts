export default {
    text: '#333',
    background: '#ebf0f1',
    icon: '#0badc2',
    unknown: '#d3dfe8',
    success: '#8ddb8d',
    warning: '#d1be65',
    failure: '#e37856',
};
