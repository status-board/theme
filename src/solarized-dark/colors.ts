export default {
    text: '#90a2a4',
    textHighlight: '#eee8d5',
    background: '#073642',
    icon: '#0badc2',
    unknown: '#495b71',
    success: '#859900',
    warning: '#b58900',
    failure: '#dc322f',
};
