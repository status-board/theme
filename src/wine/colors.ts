export default {
    text: 'hsl(6, 26%, 67%)',
    textHighlight: '#fff',
    background: '#281212',
    icon: 'hsl(0, 52%, 60%)',
    unknown: '#7e706d',
    success: '#50a3b2',
    warning: '#b87334',
    failure: '#a31c12',
};
