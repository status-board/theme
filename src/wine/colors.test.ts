import colors from './colors';

test('Colors should match snapshot', (): void => {
    expect(colors).toMatchSnapshot();
});
