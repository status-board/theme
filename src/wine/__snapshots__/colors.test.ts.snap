// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors should match snapshot 1`] = `
Object {
  "background": "#281212",
  "failure": "#a31c12",
  "icon": "hsl(0, 52%, 60%)",
  "success": "#50a3b2",
  "text": "hsl(6, 26%, 67%)",
  "textHighlight": "#fff",
  "unknown": "#7e706d",
  "warning": "#b87334",
}
`;
